lib/kommit.rb

```diff
@@ -12,6 +13,8 @@
 @chance_to_commit_on_saturday = nil
 @chance_to_commit_on_sunday = nil
 @windows_machine = RUBY_PLATFORM.match?(/mswin|mingw/)
+GH_USERNAME = ENV['GH_USERNAME']
```

Comment:
```suggestion
GH_USERNAME = `gh api user --jq '.login'`.strip
```

@42LM Thanks for contribution ❤️ WDYT about reading the username directly from the terminal though? That way there's even less setup for the user